gloo/docs/gloo_routing/virtual_services/security/basic_auth/_index.md

```diff
@@ -45,13 +57,13 @@ glooctl add route --name test-no-auth --path-prefix / --dest-name json-upstream
 {{< /tab >}}
 {{< /tabs >}} 
 
-Let's make a request to that route and make sure it works:
+Let's send a request that matches the above route to the Gloo Gateway and make sure it works:
 
 ```shell
 curl -H "Host: foo" $GATEWAY_URL/posts/1
 ```
 
-returns
```

Comment:
i kind of preferred the brevity here

Reply:
I tend to dislike using code blocks as elements of a sentence (when sentences have no meaning without them) and usually prefer referencing them explicitly in the text. It's just a matter of taste and it varies case by case, but something like this looks a bit lazy and somehow off to me:

![image](https://user-images.githubusercontent.com/16148461/66322636-71c45c80-e8f0-11e9-9142-7aa96201d96d.png)

I can shorten to `The above command should return:`